package.json

```diff
@@ -14,52 +14,53 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "2.4.9",
-    "@angular/compiler": "2.4.9",
-    "@angular/core": "2.4.9",
-    "@angular/forms": "2.4.9",
-    "@angular/http": "2.4.9",
-    "@angular/platform-browser": "2.4.9",
-    "@angular/platform-browser-dynamic": "2.4.9",
-    "@angular/platform-server": "2.4.9",
-    "@angular/router": "3.4.9",
-    "core-js": "2.4.1",
-    "rxjs": "5.2.0",
+    "@angular/common": "^2.4.0",
```

Comment:
I'm not a fan of allow "floating" versions. I've personally had too many instances when a minor update broke my code. I follow the same approach I would with Java -- specifying the exact version that works.